stream/stream.js

```diff
@@ -2,161 +2,147 @@
 ;(function() {
 "use strict"
 /* eslint-enable */
-
-var guid = 0, HALT = {}
-function createStream() {
-	function stream() {
-		if (arguments.length > 0 && arguments[0] !== HALT) updateStream(stream, arguments[0])
-		return stream._state.value
+Stream.SKIP = {}
+Stream.scan = scan
+Stream.merge = merge
+Stream.combine = combine
+Stream.scanMerge = scanMerge
+Stream["fantasy-land/of"] = Stream
+
+let warnedHalt = false
+Object.defineProperty(Stream, "HALT", {
+	get: () => {
```

Comment:
This is the only arrow function I've seen in this file, the other `defineProperty` uses `get: function...`

Is this intentional?

Reply:
Whoops. No, well spotted 😉